code/script.js

```diff
@@ -1,47 +1,195 @@
+
+  document
+    .getElementById('cssBtn')
+    .addEventListener('click', () => nextQuestion("CSS"))
+
+  document
+    .getElementById('jsBtn')
+    .addEventListener('click', () => nextQuestion("Javascript"))
+}
+
+const showMenu = (type) => {
+  questionNumber++
+  botReply(
+    `Oh interesting, that you picked ${type}.`
+  )
+
+  if (type === 'Html!') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Why did you pick it?</option>
+        <option value="So easy to get started">So easy to get started</option>
+        <option value="It's the base of everything!">It's the base of everything!</option>
+      </select>
+    `
+  } else if (type === "CSS") {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Why did you pick it?</option>
+        <option value="Styling is the best! Design over everything!">Styling is the best! Design over everything!</option>
+        <option value="Very creative and shows individuality">Very creative and shows individuality</option>
+      </select>
+    `
+
+  } else if (type === "Javascript") {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Why did you pick it?</option>
+        <option value="Advanced skill">Advanced skill</option>
+        <option value="Making every page interactive">Making every page interactive</option>
+      </select>
+    `
+  }
+
+  const select = document.getElementById('select')
+  select.addEventListener('change', () => nextQuestion(select.value))
+}
+
+const showLike = (like) => {
+  questionNumber++
+  botReply(`O cool! Find out what i felt about it.`)
+  inputWrapper.innerHTML = `
+    <button id="Html/CSS">Html/CSS</button>
+    <button id="Javascript">Javascript</button>
+  `
+
+  document
+    .getElementById('Html/CSS')
+    .addEventListener('click', () => nextQuestion('Html/CSS'))
+
+  document
+    .getElementById('Javascript')
+    .addEventListener('click', () => nextQuestion('Javascript'))
 }
 
-// Set up your eventlisteners here
+const showFeeling = (feeling) => {
+  questionNumber++
+  let gif
+  if (feeling === 'Html/CSS') {
+    gif = '<iframe src="https://giphy.com/embed/XreQmk7ETCak0" width="200" height="200" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/retro-thumbs-up-XreQmk7ETCak0"></a></p>'
+
+  } else {
+    gif = '<iframe src="https://giphy.com/embed/ySpxjJmsq9gsw" width="200" height="200" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/ySpxjJmsq9gsw"></a></p>'
+  }
+
+  botReply(
+    `This is how i felt about it: ${gif} Are you surprised? `
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="restart">NO</button>
+    <button id="confirm">YES</button>
+  `
+  document.getElementById('restart').addEventListener('click', () => {
+    location.reload()
+    return false
```

Comment:
Maybe change this to just another message since it's an answer to "Are you surprised?"